web/src/main/java/org/fao/geonet/kernel/harvest/harvester/ogcwxs/Harvester.java

```diff
@@ -192,7 +193,7 @@ public OgcWxSResult harvest() throws Exception {
         // Try to load capabilities document
 		this.capabilitiesUrl = getBaseUrl(params.url) +
         		"SERVICE=" + params.ogctype.substring(0,3) +
-        		"&VERSION=" + params.ogctype.substring(3) +
+        		"&ACCEPTVERSIONS=" + params.ogctype.substring(3) +
```

Comment:
This will work for SOS service (also for WFS - to check) but not for CSW or WMS. Should depend on service type ?